### ISSUE TITLE
[question] encoding problem with translation

### ISSUE THREAD

**Issue by lingjiameng** [bvschaik/julius#181]
Firstly, I want to make a  Chinese translation patch. But as I dig deeper into this problem, I found it's not so easy. Not readable language file ( `.eng`), rendering font from spliced bitmap file ( `.sig`, `.555`), and the processing unit of all characters is `Bytes` or `uint8_t`. 
The major files involved in this translation problem are `image.c` `font.c` and `text.c`.
That's all I founded, maybe some mistakes ?
So my questions are listed as below:
 1. Can this problem be solved with less impact (such as only change the files listed above )? 
 2. **Is it a possible solution for this problem by changing processing unit to `utf-8` ?**
Changing processing unit to utf-8 makes it's possible for any language translation.

**Comment by bvschaik** [comment]
The game was made 20 years ago with just Latin1 in mind as character set. Since those are the (font) graphics we have to work with in Julius, it's indeed not easy to change that. For Cyrillic, I've implemented side-loading extra graphics for the Cyrillic letters, but I don't think we want to go that way for Chinese, since Cyrillic has only a few dozen characters but Chinese has way more.

To answer your questions:
1. I guess not, see explanation above.
2. I fear not. Handling UTF-8 is non-trivial, as I found out when supporting accented characters in filenames.
